testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/deployment/OidcWithDeploymentConfigTest.java

```diff
@@ -398,6 +441,29 @@ public void testCorsRequestWithEnableCorsWithInvalidOrigin() throws Exception {
         }
     }
 
+    @Test
+    public void testOpenIDScopeValue() throws Exception {
+        deployer.deploy(OPENID_SCOPE_APP);
+        super.testOpenIDScope();
```

Comment:
Need to make sure we undeploy the application if this is the last test case that's making use of it.